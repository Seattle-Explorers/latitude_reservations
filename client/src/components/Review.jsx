import React from 'react';
import PropTypes from 'prop-types';
import { IoIosStar } from 'react-icons/io';

class Review extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
    };
  }

  render() {
    const { review } = this.props;

    return (
      <div className="star-review">
        <IoIosStar />
        <h2>{review}</h2>
      </div>

    );
  }
}

Review.propTypes = {
  review: PropTypes.number.isRequired,
};

export default Review;
