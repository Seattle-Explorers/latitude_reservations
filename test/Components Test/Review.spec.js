import React from 'react';
import { shallow, configure } from 'enzyme';
import Review from '../../client/src/components/Review.jsx';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Review Component', () => {
  it('should render correctly when given a valid listing ID', () => {
    const component = shallow(<Review review={4} />);

    expect(component).toMatchSnapshot();
  });
});
