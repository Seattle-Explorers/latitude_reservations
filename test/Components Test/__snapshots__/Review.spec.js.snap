// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review Component should render correctly when given a valid listing ID 1`] = `ShallowWrapper {}`;
